packages/nvidia_nat_nemo_customizer/src/nat/plugins/customizer/dpo/trainer_adapter.py

```
@@ -0,0 +1,550 @@
+                backend=ref.backend,
+                status=status,
+                progress=progress,
+                message=message,
+                metadata={
+                    "job_id": job_id,
+                    "nemo_status": job_status.status,
+                    "output_model": self._job_output_models.get(ref.run_id),
+                },
+            )
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)